main/templates/citation/export.html

```diff
@@ -22,6 +22,23 @@
     </span>
   {% endfor %}
   {% for format in available_serialization_formats %}
+    {% if format == "bibxml" and xml2rfc_urls %}
+      {% for _, url, _ in xml2rfc_urls|slice:":1" %}
```

Comment:
Just curious. Won't

```
xml2rfc_urls|slice:":1"
```

always return one item? In other words, is the `for`-loop necessary?

Reply:
Yes, it is intended to return a single item.
But the other way to do this with [django-template](https://docs.djangoproject.com/en/4.2/topics/templates/) is `xml2rfc_urls.0` which (IMO) didn't look nice.
Note this will also result in something like `xml2rfc_urls.0.1 down the line to get the URL.
I am happy to change if there's a better way to represent it.